webserver-darwin.sh

```diff
@@ -1,36 +1,54 @@
-#!/usr/bin/zsh
+#!/bin/bash
 
-echo "brew should be installed"
+# Install homebrew
+  echo "brew should be installed"
 if ( which brew )
 then
   echo "brew already installed"
 else
   echo "installing brew"
   /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
 fi
-echo "git should be installed"
+
+# Install git
+  echo "git should be installed"
 if ( which git )
 then 
   echo "git already installed"
 else 
   echo "installing git"
   brew install git
 fi
-echo "nano should be installed"
+
+#install nano
+  echo "nano should be installed"
 if ( which nano )
 then
   echo "nano already installed"
 else
   echo "installing nano"
   brew install nano 
 fi
- echo "installing multipass on $(uname)"
-  if [ "$(uname)" = "Darwin" ]
+
+# Install multipass
+  echo "installing multipass on $(uname)"
+if [ "$(uname)" = "Darwin" ]
```

Comment:
Why are we checking "if darwin" if the the script is called webserver-darwin.sh and if we have good instructions in the README.md?

Reply:
Are you saying I shouldn't check?